### ISSUE TITLE
Plugin-Loader Python was not found

### ISSUE THREAD

**Issue by Pfeifenjoy** [phuang/totem-plugin-airplay#1]
I tried to install the plugin, therefore I have installed totem, python, GObject and DBUS.
I also have a working AVAHI stack.

However when I want to active the plugin it always tells me that the plugin loader
for python was not found.

Do you know how to fix this?

**Comment by newptcai** [comment]
Exactly the same problem. Have you found any walk around?

**Comment by DAC324** [comment]
Apparently, this package is not maintained any more. 
There are a few other errors in the code that indicate that the plugin will not work with Python 3.

**Comment by lauzadis** [comment]
Are there any alternatives to this software?

**Comment by tototo31** [comment]
https://github.com/FD-/RPiPlay This worked for me - granted its not a totem plugin

**Comment by msangel** [comment]
Of course, this does not work. This plugin is 10 years old and it's not updated. Now even the python v2 is obsolete.
